### ISSUE TITLE
Cannot edit Feed-Licenses doc to add a new copyrght

### ISSUE THREAD

**Issue by Bro66666** [public-transport/transitous#146]
I need to add a new copyroght for a feed, but the link to edit  the document located at

https://public-transport.github.io/transitous/licenses/

does not work for me. It leads to a page that says _Not Found_.

I also cannot locate the page in the Transitous main repository.

Can you please inform where this document is located and maybe correct the link?

**Comment by PartTimeDataScientist** [comment]
The licenses are automatically extracted from the data in the GTFS feeds. The related markdown file is generated [here](https://github.com/public-transport/transitous/commit/372894763f6be14c44f3ebb975e82783635b9c36) and further processed by mkdocs in a later stage...

**Comment by PartTimeDataScientist** [comment]
It probably makes sense to add that as a remark above the licenses page to avoid future confusion. I can take care of that later...

**Comment by Bro66666** [comment]
> The licenses are automatically extracted from the data in the GTFS feeds. The related markdown file is generated [here](https://github.com/public-transport/transitous/commit/372894763f6be14c44f3ebb975e82783635b9c36) and further processed by mkdocs in a later stage...

The problem is I have found several instances where the license is not within the feed, you have to read the actual web page that hosts it.

For example:

https://www.fgv.es/transparencia/index.php

**Comment by Bro66666** [comment]
Right. I see the copyright holders for the feeds I was worried about. Thanks @PartTimeDataScientist .

The links is still confusing though. Maybe it should be removed?

**Comment by PartTimeDataScientist** [comment]
> Which link are you referring to?
> The License page can't be removed, it is the bare minimum that we need to do to fulfill the license requirements of some feeds.

I think the "edit" link on the top of the page is misleading as this page cannot an should not be manually edited. I'll try to get rid of it (should be possible using a mkdocs plug-in and put a not at the top of the page that the licenses page is automatically generated...

![edited_Screenshot_20240316-125828.png](https://github.com/public-transport/transitous/assets/30908795/2b260f14-e027-40dc-96bd-c37651849dd3)

**Comment by PartTimeDataScientist** [comment]
Why would you use two different site-generators in the project? Wouldn't it be cleaner to then get rid of mkdocs completely and also use Hugo for everything? It has the beautiful [Docsy](https://example.docsy.dev/) Theme which should serve both use cases well...

**Comment by PartTimeDataScientist** [comment]
> I don't think it really makes sense to combine the public representation and documentation into one theme.
> The goals for them are just very different.

Granted although I slightly disagree. IMHO e.g. mkdocs-materiel and even more so the Docsy theme is flexible enough to produce a beautiful overall website *and* nice documentation from the same theme.

Regardless of this discussion it still keeps the overall tech stack simpler if you use the same generator for both.
